JSDemos/Demos/PivotGrid/StandaloneFieldChooser/description.md

```diff
@@ -1 +1,14 @@
-The standalone field chooser is&nbsp;the&nbsp;PivotGridFieldChooser component that shares the data source with the PivotGrid and has three different layouts. Active layout changes can be&nbsp;applied to&nbsp;the PivotGrid in&nbsp;instant or&nbsp;on-demand mode.
\ No newline at end of file
+Along with a field chooser [integrated into the PivotGrid](https://js.devexpress.com/Demos/WidgetsGallery/Demo/PivotGrid/IntegratedFieldChooser/), DevExtreme includes a standalone [PivotGridFieldChooser](/Documentation/ApiReference/UI_Components/dxPivotGridFieldChooser/) component. Unlike its integrated counterpart, the standalone field chooser can remain visible on the page at all times.
```

Comment:
```suggestion
DevExtreme also includes a standalone [PivotGridFieldChooser](/Documentation/ApiReference/UI_Components/dxPivotGridFieldChooser/) component with the [integrated](https://js.devexpress.com/Demos/WidgetsGallery/Demo/PivotGrid/IntegratedFieldChooser/) field chooser. Unlike its integrated counterpart, the standalone field chooser can remain visible on the page at all times.
```

Reply:
> a standalone [PivotGridFieldChooser](/Documentation/ApiReference/UI_Components/dxPivotGridFieldChooser/) component with the [integrated](https://js.devexpress.com/Demos/WidgetsGallery/Demo/PivotGrid/IntegratedFieldChooser/) field chooser

The standalone field chooser doesn't include the integrated field chooser. It's DevExtreme that includes a standalone field chooser and a field chooser integrated into the PivotGrid.

Reply:
Why not just say 'DevExtreme also includes a standalone [PivotGridFieldChooser](/Documentation/ApiReference/UI_Components/dxPivotGridFieldChooser/) component'?

Reply:
We have two different demos for the integrated and standalone field choosers. Those who enter our website from the standalone field chooser demo will be confused by this "also". I quess the following version would work:

```suggestion
DevExtreme includes a standalone [PivotGridFieldChooser](/Documentation/ApiReference/UI_Components/dxPivotGridFieldChooser/) component. Unlike its [integrated counterpart](https://js.devexpress.com/Demos/WidgetsGallery/Demo/PivotGrid/IntegratedFieldChooser/), the standalone field chooser can remain visible on the page at all times.
```

Reply:
Ok